prospector/config/datatype.py

```diff
@@ -1,11 +1,11 @@
-from os import pathsep
+import re
 
 from setoptconf.datatype import Choice
 
 
 class OutputChoice(Choice):
     def sanitize(self, value):
-        splitted_value = value.split(pathsep)
-        output_format, output_targets = splitted_value[0], splitted_value[1:]
+        parsed = re.split(r'[;:]', value)
+        output_format, output_targets = parsed[0], parsed[1:]
```

Comment:
I think this raises another issue least if split by `':'`  is used on windows and this is still supposed to work with absolute paths as `-o xunit:C:\test_results\test.xml` will behave unexpectedly i think (not sure how this is used elsewhere)